odoo/addons/base/tests/test_mail.py

```diff
@@ -86,6 +86,30 @@ def test_html(self):
         for attr in ['javascript']:
             self.assertNotIn(attr, sanitized_html, 'html_sanitize did not remove enough unwanted attributes')
 
+    def test_outlook_mail_sanitize(self):
+        case = """
+        <div class="WordSection1">
+        <p class="MsoNormal">Here is a test mail<o:p></o:p></p>
+        <p class="MsoNormal"><o:p>&nbsp;</o:p></p>
+        <p class="MsoNormal">With a break line<o:p></o:p></p>
+        <p class="MsoNormal"><o:p>&nbsp;</o:p></p>
+        <p class="MsoNormal"><o:p>&nbsp;</o:p></p>
+        <p class="MsoNormal">Then two<o:p></o:p></p>
+        <p class="MsoNormal"><o:p>&nbsp;</o:p></p>
+        <div>
+        <div style="border:none;border-top:solid #E1E1E1 1.0pt;padding:3.0pt 0in 0in 0in">
+        <p class="MsoNormal"><b>From:</b> Mitchell Admin &lt;dej@gmail.com&gt;
+        <br>
+        <b>Sent:</b> Monday, November 20, 2023 8:34 AM<br>
+        <b>To:</b> Eteil DJOUMATCHOUA &lt;dej@gmail.com&gt;<br>
+        <b>Subject:</b> test (#23)<o:p></o:p></p>
+        </div>
+        </div>
+        """
+
+        html = html_sanitize(case)
+        self.assertEqual(re.search(r'</?o:.*?>', html), None)
```

Comment:
We're still not testing the actual output.
It could very well be an empty document.

I'm saying we can just copy the exact output. So we know at a glance that it's valid.

Reply:
Oh okay, I didn't understand that like that. I add it :)